### ISSUE TITLE
Error when adding parameterized query parameters panel to dashboard

### ISSUE THREAD

**Issue by rbasralian** [deephaven/web-client-ui#1366]
Whenever I add a parameterized query's parameters panel to a dashboard, I get the following error (running on `1.20230511.065 (0e5fcde6)`):

![image](https://github.com/deephaven/web-client-ui/assets/9516585/bd27260a-164c-4cdb-b320-26d35b6077cb)

```
LogProxy.js:87 TypeError: Cannot read properties of undefined (reading 'getKnownConfigs')
    at cs.getQueryNameForSerial (QueryUtils.ts:144:8)
    at new gn (ParameterizedQueryPanel.jsx:130:18)
    at fMe (react-dom.production.min.js:135:172)
    at nce (react-dom.production.min.js:181:146)
    at $Me (react-dom.production.min.js:269:427)
    at RMe (react-dom.production.min.js:250:347)
    at BKe (react-dom.production.min.js:250:278)
    at gO (react-dom.production.min.js:250:138)
    at AK (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Sx (react-dom.production.min.js:122:325)
    at sMe (react-dom.production.min.js:123:61)
    at mm (react-dom.production.min.js:122:428)
    at TOe (react-dom.production.min.js:292:101)
    at Nqe (react-dom.production.min.js:73:352)
```

However, it works after refreshing the page:

![image](https://github.com/deephaven/web-client-ui/assets/9516585/00c3a38d-dc38-4057-9969-c76e05066efa)

**Comment by mofojed** [comment]
@rbasralian can you try this on the latest? Pretty sure this issue has been fixed

**Comment by mofojed** [comment]
Should be fixed in `1.20230511.080`: https://deephaven.atlassian.net/browse/DH-14928
With PR: https://github.com/deephaven-ent/iris/pull/462
